RetailItem.java

```diff
@@ -0,0 +1,28 @@
+package ItemPackage;
+
+public class RetailItem {
+    
+    //fields
+    private String description="";
+    private int unitsOnHand = 0;
+    private double price = 0.0;
+    
+    //constructor accepts values for all fields
+    public RetailItem(int unitsOnHand, double price, String description){
+        this.unitsOnHand = unitsOnHand;
+        this.price = price;
+        this.description = description;
+    }
+    
+    //Mutators
+    public void setUnitsOnHand(int unitsOnHand){this.unitsOnHand = unitsOnHand;}
+    public void setPrice(double price){this.price = price;}
+    public void setDescription(String descr){this.description = descr;}
```

Comment:
Nothing wrong here but in terms of style you should keep the pattern you have above and keep the argument the same spelling as the field.